.eslintcache

```diff
@@ -1 +1 @@
":"3","/Users/deuex/Desktop/todo/to-do/react-todo-list-master/src/components/Todo.js":"4","/Users/deuex/Desktop/todo/to-do/react-todo-list-master/src/components/TodoForm.js":"5"},{"size":197,"mtime":1615742134000,"results":"6","hashOfConfig":"7"},{"size":211,"mtime":1615742134000,"results":"8","hashOfConfig":"7"},{"size":1698,"mtime":1654623205331,"results":"9","hashOfConfig":"7"},{"size":799,"mtime":1654620169489,"results":"10","hashOfConfig":"7"},{"size":1192,"mtime":1654620122875,"results":"11","hashOfConfig":"7"},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},"cf1dpa",{"filePath":"15","messages":"16","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},{"filePath":"19","messages":"20","errorCount":0,"warningCount":2,"fixableErrorCount":0,"fixableWarningCount":0,"source":"21"},{"filePath":"22","messages":"23","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"14"},"/Users/deuex/Desktop/todo/to-do/react-todo-list-master/src/index.js",[],["24","25"],"/Users/deuex/Desktop/todo/to-do/react-todo-list-master/src/App.js",[],"/Users/deuex/Desktop/todo/to-do/react-todo-list-master/src/components/TodoList.js",[],"/Users/deuex/Desktop/todo/to-do/react-todo-list-master/src/components/Todo.js",["26","27"],"import React, { useState } from 'react';\nimport TodoList from './TodoList';\nimport { RiCloseCircleLine } from 'react-icons/ri';\n\nfunction Todo({ todos, completeTodo, removeTodo }) {\n \n\n    return todos.map((todo, index) => (\n        <div className={todo.isComplete ? 'todo-row complete' : 'todo-row'} key={index}>\n            <input type=\"checkbox\" onChange={()=> {completeTodo(todo.id)}}/>\n            <div key={todo.id} onClick={() => completeTodo(todo.id)}>\n                {todo.text}\n            </div>\n            <div className='icons'>\n                <RiCloseCircleLine\n                    onClick={() => removeTodo(todo.id)}\n                    className='delete-icon'\n                />\n                \n                \n            </div>\n        </div>\n    ));\n};\n\nexport default Todo;\n","/Users/deuex/Desktop/todo/to-do/react-todo-list-master/src/components/TodoForm.js",[],{"ruleId":"28","replacedBy":"29"},{"ruleId":"30","replacedBy":"31"},{"ruleId":"32","severity":1,"message":"33","line":1,"column":17,"nodeType":"34","messageId":"35","endLine":1,"endColumn":25},{"ruleId":"32","severity":1,"message":"36","line":2,"column":8,"nodeType":"34","messageId":"35","endLine":2,"endColumn":16},"no-native-reassign",["37"],"no-negated-in-lhs",["38"],"no-unused-vars","'useState' is defined but never used.","Identifier","unusedVar","'TodoList' is defined but never used.","no-global-assign","no-unsafe-negation"]
```

Comment:
We don't need to commit generated files. Please add such files to .gitignore and avoid committing them.

Reply:
okay got it